### ISSUE TITLE
Update request: gajim 1.7.3 → 1.8.0

### ISSUE THREAD

**Issue by toastal** [NixOS/nixpkgs#235394]
- Package name:
- Latest released version: 1.8.0

- Current version on the unstable channel: 1.7.3
- Current version on the stable/release channel: 1.7.3

- [x] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

**Notify maintainers**

@abbradar @7c6f434c

-----

Note for maintainers: https://gajim.org/post/2023-05-27-gajim-1.8.0-released/

**Comment by toastal** [comment]
basic version bump was not successful

```diff
diff --git i/pkgs/applications/networking/instant-messengers/gajim/default.nix w/pkgs/applications/networking/instant-messengers/gajim/default.nix
index bc3269721b7..8d7d3b139ba 100644
--- i/pkgs/applications/networking/instant-messengers/gajim/default.nix
+++ w/pkgs/applications/networking/instant-messengers/gajim/default.nix
@@ -22,11 +22,11 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "gajim";
-  version = "1.7.3";
+  version = "1.8.0";
 
   src = fetchurl {
     url = "https://gajim.org/downloads/${lib.versions.majorMinor version}/gajim-${version}.tar.gz";
-    hash = "sha256-t8yzWfdsY8pXye7Dn5hME0bOHgf+MzuyVY3hweXc0xg=";
+    sha256 = "EgH8mt0am2l9z4csGHH6rpLqTzFiBRzOPB4NCEP8TUM=";
   };
 
   format = "pyproject";
```

**Comment by DeeUnderscore** [comment]
Closed via #240066
